content/docs/concepts/csidriver/features/powerstore.md

```diff
@@ -768,3 +768,30 @@ To configure how often driver checks for changed capacity set `storageCapacity.p
 The CSI PowerStore driver supports the provisioning of Metro volumes. The process and details of how to provision and use Metro volumes can be found [here](../../../replication/high-availability).
 
 Please note that the Metro feature does not require the deployment of the replicator sidecar or the replication controller.
+
+## Shared NFS
+Shared NFS utilizes native NFS features to support large-scale ReadWriteMany (RWX) volumes through the access mode, enabling efficient shared storage across multiple consumers. It follows a client-server model, with a node serving as an NFS server—and potentially a client as well.
+- **Scalability and Flexibility**: Offers enhanced scalability and flexibility compared to traditional NFS, which relies on a single dedicated server.
+- **Centralized File Management**: Allows remote files to be accessed as if they were local, simplifying file management and reducing duplication for more efficient storage use.
+- **NFSv4 Recommendation**: For optimal performance and compatibility, NFSv4 is recommended.
```

Comment:
Are we starting the release of this improvements with an explicitly support of v4? Is it a recommendation or required as the first incarnation of the feature? Is there any reason to support the older versions?

Reply:
I believe nfsv3 also works. But recommended to have nfsv4 for better performance. I will clarify.

Reply:
updated.